package-res/resources/web/prompting/PromptPanel.js

```diff
@@ -642,7 +645,18 @@ define(['cdf/lib/Base', 'cdf/Logger', 'dojo/number', 'dojo/i18n', 'common-ui/uti
          */
         parameterChanged: function (param, name, value) {
           if (this.onParameterChanged) {
-            this.onParameterChanged(name, value);
+            if (typeof this.onParameterChanged === 'function') {
+              this.onParameterChanged(name, value);
+            } else {
+              var paramCallback = this.onParameterChanged[name];
+              if (paramCallback) {
+                if (typeof paramCallback === 'function') {
+                  paramCallback(name, value); 
+                } else {
+                  console.log("The callback should be a function");
```

Comment:
This message doesn't tell me much of what is going on. Maybe something like "The parameterChanged callback for 'name' is not a function"